wiki/javascripts/eventTypes.js

```diff
@@ -0,0 +1,266 @@
+var eventTypes = [
+    {
+      "number":1,
```

Comment:
Do we need any of the javascripts files anymore if we don't upload to GitHub pages?

Reply:
This file is no more needed. Hence removed this file